CNAME

```diff
@@ -0,0 +1 @@
+mavis.ml
```

Comment:
Not sure what this file is for, please remove it for this PR.

Reply:
Sorry for the CNAME file, I was experimenting something with my DNS server and Github custom domain feature, which results in all this